deepchem/feat/bio_seq_featurizer.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import deepchem as dc
+try:
+    import pysam
+except ImportError:
+    print("Error: Unable to import pysam. Please make sure it is installed.")
+from deepchem.feat import Featurizer
+
+class SAMFeaturizer(Featurizer):
+    """
+    This class extracts Query Name, Query Sequence, Query Length, Reference Name, 
+    Reference Start, CIGAR and Mapping Quality of the alignment in the SAM file.
```

Comment:
Expand the docstring here. What is the type of input, what is the type of output?